src/HookDispatcher.php

```diff
@@ -60,15 +60,24 @@ class HookDispatcher
      */
     private $hooks = [];
 
+    /**
+     * Module
+     *
+     * @var Ps_Facetedsearch
+     */
+    private $module;
+
     /**
      * Init hooks
      *
      * @param Ps_Facetedsearch $module
      */
     public function __construct(Ps_Facetedsearch $module)
     {
+        $this->module = $module;
```

Comment:
I don't clearly understand what does it change to use a property instead of a local variable here: what are the side effects?